src/core/BufferAttribute.js

```diff
@@ -17,23 +17,79 @@ function BufferAttribute( array, itemSize, normalized ) {
 
 	this.name = '';
 
-	this.array = array;
-	this.itemSize = itemSize;
+	this._array = array;
+	this._itemSize = itemSize;
 	this.count = array !== undefined ? array.length / itemSize : 0;
-	this.normalized = normalized === true;
+	this._normalized = normalized === true;
 
 	this.dynamic = false;
 	this.updateRange = { offset: 0, count: - 1 };
 
 	this.version = 0;
+	this.version2 = 0;
```

Comment:
Maybe `versionVAO`? This will indicate that the property is used in `VAO` context.

Reply:
Renamed. Thanks